config/browser-list-config.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+// See CONTRIBUTING.md for a list of browsers included in 'last 2 versions'
+const LAST_2_IE_9_UP = [
+  'last 2 versions',
+  'Explorer >= 9'
+];
+
+const LAST_2_IE_8_UP = [
+  'last 2 versions',
+  'Explorer >= 8'
+];
+
+module.exports = {
+  LAST_2_IE_9_UP,
+  LAST_2_IE_8_UP
```

Comment:
What do you think about renaming these to be extra clear? Something like, `LAST_2_PLUS_IE_8_AND_UP`. They're already a disaster to type without code completion, what's two more words?

Reply:
Yeah, I guess it's not much worse now. I'll make that change here and in cfgov-refresh.